### PR TITLE
Add `echo` command

### PR DESCRIPTION
**Pull request by aluminiumgeek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Refer to `random.rb` as an example.
 - g - I'm Feeling Lucky!
 - help - Show help message.
 - hey - Hey!
+- echo - Echoes your text
```
Comment (eugenzyx) [inline README.md]: Either put a period or remove it from everywhere in commands list please!

Comment (aluminiumgeek) [inline README.md]: :white_check_mark:

File: src/main/scala/com/eugenzyx/commands/Echo.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.eugenzyx.commands
+
+import com.eugenzyx.commands.traits.Command
+
+object Echo extends Command {
+  val command = "echo"
+  val description = """
+    Returns input text.
```
Comment (eugenzyx) [inline src/main/scala/com/eugenzyx/commands/Echo.scala]: `"""Returns input text…` otherwise those 4 spaces will go to the resulting string.

Comment (aluminiumgeek) [inline src/main/scala/com/eugenzyx/commands/Echo.scala]: :white_check_mark:

**Comment by eugenzyx** [comment]
Excellent!

**Pull request merged**